src/main/java/com/amazonaws/kvstranscribestreaming/TranscribedSegmentWriter.java

```diff
@@ -77,18 +112,67 @@ public void writeToDynamoDB(TranscriptEvent transcriptEvent, String tableName) {
         }
     }
 
-    private Item toDynamoDbItem(Result result) {
+    /**
+     * This method uses Amazon Translate Client to translate the text passed as a paramater according to its
+     * source language code and target language code
+     * @param sourceLangCode This signifies the language the text is written in
+     * @param targetLangCode  This signifies the language for the text to translate to
+     * @param text This is the text to translate
+     * @return String This is the translated text
+     */
+    private String translateText(String sourceLangCode, String targetLangCode, String text){
+        long cur= System.currentTimeMillis();
+        logger.info("Starting Translation: " + text);
+        try{
+            TranslateTextRequest request = new TranslateTextRequest()
+                    .withText(text)
+                    .withSourceLanguageCode(sourceLangCode)
+                    .withTargetLanguageCode(targetLangCode);
+            TranslateTextResult translateResult = translateClient.translateText(request);
+            long diff = System.currentTimeMillis()-cur;
+            String translated = translateResult.getTranslatedText();
+            logger.info("Translation: "+ translated);
+            logger.info("Translation time (milli): " + diff);
+            return translated;
+        } catch (Exception e){
+            logger.error("Exception while translating transcript: ", e);
+            logger.info("Unable to translate. Returning untranslated text");
```

Comment:
Make no sense to use INFO and ERROR level logging statements together. Just use logger.error()